sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/AnomalyDetectionLiveTests.cs

```diff
@@ -406,5 +415,23 @@ private void ValidateIncidentRootCause(IncidentRootCause rootCause)
 
             ValidateSeriesKey(rootCause.SeriesKey);
         }
+
+        private bool AreSame(DimensionKey left, DimensionKey right)
```

Comment:
Two dimension keys are considered identical if they have the same set of key-value pairs.